api/external/applications/applications.proto

```diff
@@ -116,7 +208,22 @@ service ApplicationsService {
     option (chef.automate.api.iam.policy).resource = "retention:serviceGroups";
     option (chef.automate.api.iam.policy).action = "retention:serviceGroups:get";
   };
-
+  /*
+  UpdateDeleteDisconnectedServicesConfig
+
+  Updates config information for the periodicly running task that deletes services already marked as disconnected
+  after 'threshold'. Threshold is a string that follows elasticsearch's date math expressions.
+  This job can be disabled/enabled by setting running to false.
+
+  Example:
+  ```
+  service_groups/delete_disconnected_services/config" -d
+  '{
+    "threshold": "1d",
+    "running":true
+  }'
+  ```
```

Comment:
@kagarmoe 

> The API how-to docs are here: https://github.com/chef/automate/tree/master/components/automate-chef-io#api-documentation

I read those but didn't find anything discussing how to write API usage examples like the one above. What Kyleen has done here is format the example in approximately the syntax you'd use with the `curl` tool; in particular, `-d 'BODY_DATA'` is how you specify the request body with `curl`. This made me wonder if we had some standards on how we'd like this kind of stuff to be written, or if we _should_ have some such standards.

Reply:
https://github.com/chef/automate/issues/1915

Reply:
@danielsdeleo I misunderstood the conversation. For now, Kyleen has used the best available method for writing an example. We will add the capacity for code examples for request bodies & requests in different programming languages in future iterations.

Reply:
That sounds super awesome, thanks!